<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTemperature">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>781</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#centralWidget {
border-image: url(:/temperature.png) 0 0 0 0 stretch stretch;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTemp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/red.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTemp_2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/temp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblDateTime">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>431</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>24/02/2020 09:53</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRedGear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/setgear.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 147, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>431</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 147, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ClickableLabel" name="btnLowerBound">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>100</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/bluebar.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ClickableLabel" name="lblBack">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>9</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickableLabel" name="lblTempCtl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>100</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/temp.png);background-color: rgb(255, 150, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblStartDateTime_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>701</width>
      <height>8</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickableLabel" name="lblLowerBound_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temperature History</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="btnLowerBound_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>100</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/bluebar.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ClickableLabel" name="lblLowerBound_3">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>69</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>24 Hrs View</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="btnLowerBound_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>100</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/bluebar.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ClickableLabel" name="lblLowerBound_4">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>69</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Week View</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="btnLowerBound_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>100</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/bluebar.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ClickableLabel" name="lblLowerBound_5">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>69</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Month View</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="lblLowerBound_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>30 *C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>741</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <zorder>btnTemperature</zorder>
   <zorder>lblStartDateTime_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>lblTemp</zorder>
   <zorder>lblTemp_2</zorder>
   <zorder>lblDateTime</zorder>
   <zorder>lblRedGear</zorder>
   <zorder>btnLowerBound</zorder>
   <zorder>lblBack</zorder>
   <zorder>lblTempCtl</zorder>
   <zorder>lblLowerBound_2</zorder>
   <zorder>btnLowerBound_2</zorder>
   <zorder>lblLowerBound_3</zorder>
   <zorder>btnLowerBound_3</zorder>
   <zorder>lblLowerBound_4</zorder>
   <zorder>btnLowerBound_4</zorder>
   <zorder>lblLowerBound_5</zorder>
   <zorder>lblLowerBound_6</zorder>
   <zorder>gView</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
